packages/shared/lib/sdk/sync.ts

```diff
@@ -533,7 +533,11 @@ export class NangoAction {
         const response = await persistApi({
             method: 'POST',
             url: `/environment/${this.environmentId}/log`,
+            headers: {
+                Authorization: `Bearer ${this.nango.secretKey}`
+            },
             data: {
+                secretKey: this.nango.secretKey,
```

Comment:
Why sending in the body too?

Reply:
oups. A leftover of my first pass. Fixing